Library/Homebrew/os/mac/mach.rb

```diff
@@ -60,6 +60,14 @@ def dylib_id
     macho.dylib_id
   end
 
+  def rpaths
+    macho.rpaths
+  end
+
+  def delete_rpath(rpath)
+    macho.delete_rpath(rpath)
+  end
```

Comment:
I believe you can use `delegate` like so…
```ruby
delegate [:delete_rpath, :rpaths] => :macho
```
We use it elsewhere in the Homebrew code base.